src/esp/bindings/GfxBindings.cpp

```diff
@@ -205,6 +206,46 @@ void initGfxBindings(py::module& m) {
       .def(py::self == py::self)
       .def(py::self != py::self);
 
+  py::class_<DebugLineRender, std::shared_ptr<DebugLineRender>>(
+      m, "DebugLineRender")
+      .def(
+          "set_line_width", &DebugLineRender::setLineWidth,
+          R"(Set global line width for all lines rendered by DebugLineRender.)")
+      .def(
+          "push_transform", &DebugLineRender::pushTransform,
+          R"(Push (multiply) a transform onto the transform stack, affecting all line-drawing until popped. Must be paired with popTransform().)")
+      .def("pop_transform", &DebugLineRender::popTransform,
+           R"(See push_transform.)")
+      .def("draw_box", &DebugLineRender::drawBox,
+           R"(Draw a box in world-space or local-space (see pushTransform).)")
```

Comment:
No color for box bindings?

Reply:
draw_box takes two Vector3 and a color. This binding here just takes all the C++ function parameters as-is without explicitly declaring them all here.

Reply:
I guess this just makes it harder for python only user seeing the doc-string API reference to know the parameters. Also, does implicit conversion allow keyword (named) arguments?